ddp-workspace/projects/ddp-circadia/src/app/components/pages/home/home.component.ts

```diff
@@ -58,7 +58,11 @@ export class HomeComponent implements OnInit, AfterViewInit, OnDestroy {
       return;
     }
 
-    const yOffset = el.offsetTop - this.headerHeight;
+    const currentFontSize = +getComputedStyle(
+      document.documentElement,
+    ).fontSize.slice(0, 1);
```

Comment:
that looks a little bit tricky. What's the minimal and maximum font-size might be?

Reply:
All measurements in the application are done using `rem` unit. On desktop `1rem` is equal to `10px` (by setting `html` element `font-size` to `62.5%`). On smaller screens we reduce it down to `9px` (`font-size: 56.25%`).

Since `font-size` is dynamic, we have to get current value for it which this line is responsible for.

Reply:
This way just by changing `font-size` on `html` we can affect on whole layout without changing each element's styles.

Reply:
so when font-size is 10px the currentFontSize  will be 1, but when 9px the currentFontSize will be 9. And 
const yOffset = el.offsetTop - 10; vs const yOffset = el.offsetTop - 90;
is that expected behavior?

Reply:
Oh, correct, that my be a problem.

Thank you! I'll fix that.